.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.6.0
+    hooks:
+      - id: black
+        language_version: python3.8
+  - repo: https://github.com/pre-commit/mirrors-prettier
```

Comment:
Should we also add eslint?

Reply:
@sfc-gh-kbregula thank you for pointing this out 👍 I think we should add both `typecheck` and `eslint`, what's more we should think about adding `mypy` script. I've just tried to do it, and:

- didn't found ready pre-commit fork of `typecheck` we must do it with custom command
- there is eslint precommit fork, however it didn't discover configuration file `.eslintrc` which is not in the root directory of the project, so I must spend more time the ways how to fix it
- mypy requires using our pipenv shell and our custom mypy scripts

My proposition is to merge this PR, and step by step add more to our pre-commit script, is that ok for you?

Reply:
I'm fine with this. This is still a step forward.

Reply:
@sfc-gh-kbregula thank you for the help with pre-commit.yaml I've updated this PR with 
* isort
* typecheck
* eslint
Can you double check if these changes are ok? I've added `files` selector to every hook